requirements.txt

```diff
@@ -1,2 +1,4 @@
 requests
 python-rpm-spec
+osc
+keyring
```

Comment:
What is keyring used for?

Reply:
`osc` uses it if you use a keyring to store your credentials (e.g. `gnome-keyring`). It was not automatically pulled in when I used `pip install osc`. Not sure if we need it in the requirements.

Reply:
Then I'd say osc should pull that in, since we are not requiring that directly.

Reply:
Done.